### PR TITLE
Update chapters 1-3 of tutorial to work with OTP2

### PR DESCRIPTION
**Pull request by leonardehrenfried**
There's quite a bit of configuration that's potentially not relevant for the tutorial as it's only applicable for the Herrenberg installation but I wasn't quite sure which features should work.

Candidates for removal:

- About this page
- GeoJSON layers

### CODE REVIEW

**Review by hbruch — COMMENTED** [review]
Would be great, if the commits would be more fine-grained, so migration steps can be deduced from them. Further, a format correcting commit before doing the actual migration would be helpful.

File: 01-installing-and-running-ui/README.md [hunk 1]
```diff
@@ -153,185 +183,396 @@ In the console you will see this message `Digitransit-ui available on port 8080`
...
+const STATIC_MESSAGE_URL =
     process.env.STATIC_MESSAGE_URL ||
     '/assets/messages/message.hb.json';
 
-    const walttiConfig = require('./config.waltti.js').default;
+const walttiConfig = require('./config.waltti.js').default;
+
+const realtimeHbg = require('./realtimeUtils').default.hbg;
+const hostname = new URL(API_URL);
+realtimeHbg.mqtt = `wss://${hostname.host}/mqtt/`;
```
Comment (hbruch) [inline 01-installing-and-running-ui/README.md:205]: Step 1 of the tutorial should explain the bare minimum. No realtime service, fare service, roadworks, echarging, weather station... Only features which will be available out-of-the-box using the otp-setup of step 2.

File: 01-installing-and-running-ui/README.md [hunk 2]
```diff
@@ -153,185 +183,396 @@ In the console you will see this message `Digitransit-ui available on port 8080`
...
-            header: 'Om tjänsten',
-            paragraphs: [
-              'Den här tjänsten erbjuds av Reutlingen för reseplanering inom Reutlingen region. Reseplaneraren täcker med vissa begränsningar kollektivtrafik, promenad, cykling samt privatbilism. Tjänsten baserar sig på Digitransit-plattformen.',
-            ],
-          },
+    },
+
+    redirectReittiopasParams: true,
+
+    showTicketInformation: true,
```
Comment (hbruch) [inline 01-installing-and-running-ui/README.md]: Please remove from step 1

File: 01-installing-and-running-ui/README.md [hunk 3]
```diff
@@ -153,185 +183,396 @@ In the console you will see this message `Digitransit-ui available on port 8080`
...
+            de: "Regulär",
+        };
+    },
+    displayFareInfoTop: false,
+
+    showRouteSearch: false,
+    showNearYouButtons: false,
+
+    // adding assets/geoJson/hb-layers layers
+    geoJson: {
```
Comment (hbruch) [inline 01-installing-and-running-ui/README.md:416]: Geojson layers have not been part of step 1 before and should be removed

**Comment by leonardehrenfried** [comment]
I've worked through the tutorial again and it now works with OTP2 and Digitransit next.

I didn't touch chapter 4 as GeoJSON layers have not changed. Should we talk about the various other layers (echarging, bike parking...) there?

**Pull request merged**